src/controllers/__tests__/sessionController.test.ts

```diff
@@ -94,14 +96,16 @@ describe("SessionController#getSessionState", () => {
       bpdToken: mockBPDToken,
       myPortalToken: mockMyPortalToken,
       spidLevel: "https://www.spid.gov.it/SpidL2",
-      walletToken: mockedUser.wallet_token
+      walletToken: mockedUser.wallet_token,
+      zendeskToken: mockZendeskToken
```

Comment:
```suggestion
      zendeskToken: mockedUser.zendesk_token
```

Reply:
I tried to be as retro compatible as possible, since bpdToken and myPortalToken were added later to the user this happened also to zendeskToken. I did not put it inside the mockedUser because we test for its auto generation when not present. 

Won't fix.